samples/bluetooth/llpm/README.rst

```diff
@@ -77,12 +77,16 @@ Requirements
 
 * Two of the following nRF52-series development kit boards:
 
+  * |nRF5340DK|
```

Comment:
nRF5340PDK

Reply:
No, the shortcut is correct - it resolves to "nRF5340 PDK board (PCA10095) - see :ref:`ug_nrf5340`"